src/Tribe/RSVP.php

```diff
@@ -1003,6 +1004,28 @@ public function delete_ticket( $event_id, $ticket_id ) {
 		return true;
 	}
 
+	/**
+	 * Trigger for tribe_get_cost if there are only RSVPs
+	 * @since TBD
+	 *
+	 * @param string $cost
+	 * @param int $post_id
+	 * @param boolean $with_currency_symbol
+	 *
+	 * @return string $cost
+	 */
+	public function tribe_trigger_get_cost( $cost, $post_id, $with_currency_symbol ) {
+
+		if (
+			empty( $cost )
+			&& tribe_events_has_tickets( get_post( $post_id ) )
+		) {
+			$cost = 'Free';
```

Comment:
Do we need to translate? If not explain the reason so nobody in the future changes that.